docs/src/capi/howto/rollout-upgrades.md

```diff
@@ -0,0 +1,120 @@
+# Upgrade the Kubernetes version of a cluster
+
+This guide will walk you through the steps to rollout an upgrade for a
+Cluster API managed Kubernetes cluster. The upgrade process includes updating
+the Kubernetes version of the control plane and worker nodes.
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- A Kubernetes management cluster with Cluster API and providers installed
+  and configured.
+- A target workload cluster managed by CAPI.
+- `kubectl` installed and configured to access your management cluster.
+- The workload cluster kubeconfig. We will refer to it as `c1-kubeconfig.yaml`
+  in the following steps.
+
+Please refer to the [getting-started guide][getting-started] for instructions
+on how to set this up.
+In the following steps we will refer to the workload cluster as `c1` and its
+kubeconfig as `c1-kubeconfig.yaml`.
+
+## Check the current cluster status
+
+Before starting the upgrade, ensure your management cluster is in a healthy
+state.
+
+```
+kubectl get nodes -o wide
+```
+
+Also, verify that the workload cluster runs on the expected Kubernetes version
+
+```
+kubectl --kubeconfig c1-kubeconfig.yaml get nodes -o wide
+```
+
```

Comment:
We could add a note here to tell people to only update minor versions.

Reply:
done - thanks for all the great suggestions!